### PR TITLE
Removes all Lavaland ghost roles because according to Slovenian anyone who goes to mining as a ghost roles is 'metagaming' and should be noted and ban. Trying to argue the point with him will easily net you with a forum ban and have your argument censored from public view. Lavaland ghost roles are basically fucked now since council who dont even play the game are treating their core mechanic as a bannable offense even though they dont even play the game.

### PR DESCRIPTION
**Pull request by Stealthkibbler**
Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming. 
 Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming. 
Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming. 
Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming. 
Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming. 
Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming.
 Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming. 
Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming. 
Slovenian:  going directly for Mining base is OOC knowledge you've gained as a ghost and thus metagaming. 

:cl: Stealthkibbler
del: loads of good content removed because the council who barely play the game know nothing of the core mechanics of lavaland and its roles.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
idc to read your title but the ooc comment is perfectly valid and using ghost knowledge to find the mining base is actually metagaming

**Comment by Stealthkibbler** [comment]
Shadow noone values your opinion, all you do is meme on prs and shitpost everywhere. Fuck off.

**Comment by ghost** [comment]
@Stealthkibbler 

Actually.

I proposed lavaland rules that edit that statement. EMT seems to like them and i've recieved positive reception from other council members-Slov is the only one who hasn't commented.

I'm waiting on him and if so that can actually fix the situation here.

**Comment by ShadowDeath6** [comment]
you know you're assmad when you ad hominem someone who responds seriously to your pr :^)

**Comment by ghost** [comment]
it isn't metagame considered by the majority of admins, including a complaint that was made on Xeonfaux that was considered valid including by EMT simply because lavaland roles do need to know the terrain of the area to interact

if it was a golem respawning and rushing a cult base that is one thing, but lavaland roles do and are treated as innately knowing the location of any and all ruins and areas on lavaland simply because it's impossible for most of them aside from golems to have anything to do otherwise and they aren't like miners who have mesons

the new rules that i proposed that are just pending at the moment adress this issue

tl;dr can't meta locations and such

**Comment by BOBsHERE** [comment]
The mental gymnastics required to say that its meta to know the location of the mining base is just astounding. Just.. come on now. If you're going to make it bannable to play the role, you may as well just remove the roll and save community members undue grief over bullshit notes/bans.

**Comment by X-TheDark** [comment]
While this may be a valid complaint about rules and how they are enforced, a valid PR this is not.

**Comment by monster860** [comment]
You dont need to be a ghost to know where the minibg base is. You just need to look at a map editor.
